src/Compilers/CSharp/Test/Semantic/Semantics/RefLocalTests.cs

```diff
@@ -1,1434 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
-using Microsoft.CodeAnalysis.Test.Utilities;
-using Roslyn.Test.Utilities;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.CSharp.UnitTests
-{
-    public class RefLocalTests : CompilingTestBase
```

Comment:
They could have been not a part of the project even before the porting. Some are fairly old - a lot of tests for ref reassignment that is not supported. 
It looks like there was some effort to move these tests to Emit prior to porting, but it was not complete.  Some tests have duplicates in Emit (most of the running tests), but many compile error tests do not have duplicates and should be salvaged.